sdk/ai/azure-ai-generative/tests/evaluate/e2etests/test_evaluate_e2e.py

```diff
@@ -56,3 +56,96 @@ def test_evaluate_built_in_metrics(self, e2e_openai_api_base, e2e_openai_api_key
+        ]
+
+        from azure.ai.generative.evaluate.metrics import PromptMetric
+        custom_prompt_metric = PromptMetric.from_template(path="test_template.jinja2", name="gpt_groundedness")
+
+        with pytest.raises(Exception) as ex:
+            output_path = tmpdir + "/evaluation_output"
+
+            result = evaluate(  # This will log metric/artifacts using mlflow
+                evaluation_name="rag-chat-1",
+                data=test_data,
+                task_type="qa",
+                metrics_list=["gpt_groundedness", custom_prompt_metric],
+                model_config={
+                    "api_version": "2023-07-01-preview",
+                    "api_base": e2e_openai_api_base,
+                    "api_type": "azure",
+                    "api_key": e2e_openai_api_key,
+                    "deployment_id": e2e_openai_completion_deployment_name,
+                },
+                data_mapping={
+                    "questions": "question",
+                    "contexts": "context",
+                    "y_pred": "answer",
+                    "y_test": "truth",
+                },
+                output_path=output_path
+            )
+
+        assert "Duplicate metric name found" in str(ex.value)
+
+
+    def test_custom_metrics_name(self, e2e_openai_api_base, e2e_openai_api_key, e2e_openai_completion_deployment_name, tmpdir):
+        test_data = [
```

Comment:
maybe this test_data could be moved to a separate test data file which could be referred by different tests.

Reply:
Agreed. Will have a subsequent PR to refactor it as part of adding more tests.